Instagram/Models/Post.m

```diff
@@ -0,0 +1,55 @@
+//
+//  Post.m
+//  Instagram
+//
+//  Created by Sarah Wang on 7/7/21.
+//
+
+
+#import "Post.h"
+
+@implementation Post
+
+@dynamic postID;
+@dynamic userID;
+@dynamic author;
+@dynamic caption;
+@dynamic image;
+@dynamic likeCount;
+@dynamic commentCount;
+
+
++ (nonnull NSString *)parseClassName{
+    return @"Post";
+}
+
++ (void) postUserImage: ( UIImage * _Nullable )image withCaption: ( NSString * _Nullable )caption withCompletion: (PFBooleanResultBlock  _Nullable)completion {
```

Comment:
This spacing is super inconsistent. It is not enough for code to be correct in the "it works" sense.

```
+ (void)postUserImage:(UIImage *_Nullable)image withCaption:(NSString *_Nullable)caption withCompletion:(PFBooleanResultBlock  _Nullable)completion {
```